services/QuillLMS/spec/controllers/api/v1/app_settings_controller_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
 RSpec.describe Api::V1::AppSettingsController, type: :controller do
-  let(:user) { create(:user) }
+  let(:user) { create(:user, role: 'staff') }
```

Comment:
Should `role: staff` need to be applied to all endpoints or just `#admin_show`?

Reply:
Yes, staff authentication only applies to #admin_show, since only this action has the potential to leak user emails.